examples/cpu/plotting.cpp

```diff
@@ -47,40 +55,38 @@ int main(void){
      * it can be used for rendering text */
     fg::Font fnt;
 #ifdef OS_WIN
-    fnt.loadSystemFont("Calibri", 32);
+    fnt.loadSystemFont("Calibri");
 #else
-    fnt.loadSystemFont("Vera", 32);
+    fnt.loadSystemFont("Vera");
```

Comment:
Can't we have this as a default option for `wnd` ? This way we don't have to set it every time.